release/models/system/openconfig-system.yang

```diff
@@ -372,6 +378,24 @@ module openconfig-system {
       description
         "The amount of space currently in use on the filesystem.";
     }
+
+    container counters {
+      description
+        "Counters for tracking filesystem errors";
+        uses system-mount-point-error-counters;
+    }
+
+  }
+
+  grouping system-mount-point-error-counters {
+    description
+      "Counters for system memory errors.";
+
+    leaf io-errors {
+      type uint64;
```

Comment:
Likely rather `oc-yang:counter64`

Reply:
Thanks, fixing this and moving these under /components/component/storage/state/ with descriptions for each leaf